data-model/fdo-type/machine-annotation-service/0.3.0/schema/mas.json

```diff
@@ -0,0 +1,242 @@
+      "examples": [
+        "https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX"
+      ]
+    },
+    "ods:type": {
+      "type": "string",
+      "description": "The DOI to the FDO type of the object",
+      "pattern": "^https:\/\/doi\\.org\/[\\w\\.]+/[\\w\\.]+",
+      "examples": [
+        "https://doi.org/10.15468/1a2b3c"
+      ]
+    },
+    "schema:version": {
+      "type": "integer",
+      "minimum": 1,
+      "description": "Version of the Machine Annotation Service, https://schema.org/version"
+    },
+    "schema:name": {
+      "type": "string",
+      "description": "Name of the Machine Annotation Service as provided by the user, https://schema.org/name",
+      "examples": [
+        "GBIF Linker Service"
+      ]
+    },
+    "schema:description": {
+      "type": "string",
+      "description": "Description of the Machine Annotation Service as provided by the user, https://schema.org/description",
+      "examples": [
+        "A service that links GBIF records to a DiSSCo Digital Specimen. It creates an EntityRelationship indicating the relationship"
+      ]
+    },
+    "schema:dateCreated": {
+      "type": "string",
+      "description": "Timestamp of creation. Internally generated, https://schema.org/dateCreated",
+      "format": "date-time",
+      "examples": [
+        "2021-06-01T12:00:000Z"
+      ]
+    },
+    "ods:dateTombstoned": {
+      "type": "string",
+      "description": "Timestamp Machine Annotation Service was tombstoned and no longer active. Generated on request of the user",
+      "format": "date-time",
+      "examples": [
+        "2021-06-01T12:00:000Z"
+      ]
+    },
+    "ods:creator": {
+      "type": "string",
+      "description": "The internal user ID of the creator of the Machine Annotation Service",
+      "examples": [
+        "e2befba6-9324-4bb4-9f41-d7dfae4a44b0"
+      ]
+    },
+    "ods:containerImage": {
+      "type": "string",
+      "description": "URI of the image of the containerized application",
+      "example": [
+        "public.ecr.aws/dissco/mindat-georeferencing"
+      ]
+    },
+    "ods:containerTag": {
+      "type": "string",
+      "description": "Tag of the image",
+      "example": [
+        "sha-cb76994"
+      ]
+    },
+    "ods:TargetDigitalObjectFilter": {
+      "type": "object",
+      "description": "Filters describing the criteria that must be met in order to apply the MAS. No filters implies the MAS may run on *any* digital object. Field names are given in JSON paths, and accepted values for that field are provided as an arrays",
+      "example": [
+        {
+          "$.ods:type": [
+            "https://doi.org/21.T11148/894b1e6cad57e921764e"
+          ],
+          "$.ods:topicDiscipline": [
+            "Astrogeology",
+            "Geology",
+            "Palaeontology",
+            "Other Geodiversity",
+            "Unclassified"
+          ],
+          "$.occurrences[*].location.dwc:locality": [
```

Comment:
Same here, occurrences does not exist anymore right

Reply:
Fixed